libqpdf/QPDFValue.cc

```diff
@@ -43,3 +44,15 @@ QPDFValue::objectWarning(std::string const& warning) const
 {
     warn(QPDFExc(qpdf_e_object, "", getDescription(), 0, warning));
 }
+
+// Array methods
+
+int
+QPDFValue::size(bool type_warn, bool allow_null) const
+{
+    if (type_warn) {
+        typeWarning("array", "treating as empty");
+        QTC::TC("qpdf", "QPDFObjectHandle array treating as empty");
```

Comment:
This isn't QPDFObjectHandle anymore, so the coverage case text should be updated.

Reply:
but I don't really think this should be here -- I think it should probably stay in QPDFObjectHandle.

Reply:
All the QTC labels are tidied in the final commit.

Reply:
Missed that, thanks.